src/simulation/AliasManager.java

```diff
@@ -43,4 +49,33 @@ public String getFormattedAliases() {
         }
         return sb.toString();
     }
+
+    private String resolveFormula(String formula) {
+        int l = -1;
+        StringBuilder resolved = new StringBuilder();
+        for (int i = 0; i < formula.length(); i++) {
+            String c = String.valueOf(formula.charAt(i));
```

Comment:
Why not `char c = formula.charAt(i)`?

Reply:
Good catch, this can indeed be a `char`.